<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddVocabulary</class>
 <widget class="QDialog" name="DialogAddVocabulary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelKnownWord">
           <property name="text">
            <string>Known word</string>
           </property>
           <property name="buddy">
            <cstring>textEditKnownWord</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditKnownWord"/>
         </item>
         <item>
          <widget class="QLabel" name="labelUnknownWord">
           <property name="text">
            <string>Unknown Word</string>
           </property>
           <property name="buddy">
            <cstring>textEditUnkownWord</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditUnkownWord"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelAddedVoc">
           <property name="text">
            <string>Added Vocabulary</string>
           </property>
           <property name="buddy">
            <cstring>listWidgetAddedVoc</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidgetAddedVoc"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushBtnDeleteAddedVoc">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtnSaveVocList">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
